src/form/fields/PasswordField.test.tsx

```
@@ -1,10 +1,26 @@
 import { SchemaProvider } from '../providers/SchemaProvider';
 import { SuggestionContext } from '../providers/SuggestionRegistryProvider';
 import { ROOT_PATH } from '../utils';
 import { PasswordField } from './PasswordField';
 
+const StatefulSuggestionProvider = ({
+  children,
+  getProviders,
+}: {
+  children: ReactNode;
+  getProviders: jest.Mock;
+}) => {
```

ESLint

[error] Replace `⏎··children,⏎··getProviders,⏎}:·{⏎··children:·ReactNode;⏎··getProviders:·jest.Mock;⏎` with `·children,·getProviders·}:·{·children:·ReactNode;·getProviders:·jest.Mock·`

(prettier/prettier)